README.md

```diff
@@ -135,6 +135,23 @@ sslCipherSuite | The prioritized list of allowed SSL/TLS ciphers. For more infor
 
 Similar to configuring the server, the client configuration file is located at `/etc/opt/omi/conf/omicli.conf`.
 
+### Collecting Performance Metrics using OMI/OMI Provider on docker container or virtual machine
+
+OMI/OMI provider can work on containers and machines, just make sure you know following notes:
+- OMI/OMI provider can collect virtual machine performance metrics even in a normal container.
```

Comment:
You can have two sub-sessions:
- Virtual machine performance metrics  
- Docker container performance metrics